docs/user-guide.md

```diff
@@ -1930,9 +1930,32 @@ shown above.<br>
 <br>
 If you are writing to a local file system, the
 directory does not have to exist, but initial backups will fail if the
-destination is not available or writeable.
+destination is not available or writeable.<br>
 <br>
-If you are writing to a gcsfuse bucket, the /gcsfuse must be used as parameter.</td>
+If you are writing to a NFS share, the NFS share directory uid and gid should be configured to match your ownership needs. The NFS option requires the --nfs-backup-config option.</td>
```

Comment:
Please wrap the line here.  How should a user configure GID and UID of the mountpoint location?

Reply:
"the NFS share directory uid and gid should be configured to match your ownership needs";  I'm having trouble following here.  Where are UID and GID configured?  What value should they be set to for backups to work successfully?